install_eic.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+#!/bin/bash
+set -ex
+
+VERSION="1.1.11"
+DOWNLOAD_URL="https://github.com/aws/aws-ec2-instance-connect-config/archive/${VERSION}.tar.gz"
+
+curl -L -o instance-connect.tar.gz "${DOWNLOAD_URL}"
+TEMP_DIR=$(mktemp -d)
+tar xzf instance-connect.tar.gz -C "$TEMP_DIR"
+
+for file in ${TEMP_DIR}/aws-ec2-instance-connect-config-${VERSION}/src/bin/eic_* ; do
+    chmod +x $file
+    mv $file /usr/local/bin
```

Comment:
`/usr/local/bin` -> `/usr/local/bin/`, if for some reason `/usr/local/bin` does not exist, then you create bin as file and move  not to the directory